.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
These `.idea/` files are user-agnostic JetBrains IDE configuration files committed with [these `JetBrains.gitignore` entries](https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore) excluded via local global gitignore. I generally try to commit these to share some settings but not opposed to excluding all of them if you're opposed.